spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/feign/AnnotatedParameterProcessor.java

```diff
@@ -35,14 +37,15 @@
 	 */
 	Class<? extends Annotation> getAnnotationType();
 
-	/**
-	 * Process the annotated parameter.
-	 *
-	 * @param context    the parameter context
-	 * @param annotation the annotation instance
-	 * @return whether the parameter is http
-	 */
-	boolean processArgument(AnnotatedParameterContext context, Annotation annotation);
+    /**
+     * Process the annotated parameter.
+     *
+     * @param context    the parameter context
+     * @param annotation the annotation instance
+     * @param method     the method that contains the annotation
+     * @return whether the parameter is http
+     */
+    boolean processArgument(AnnotatedParameterContext context, Annotation annotation, Method method);
```

Comment:
Since this is not a backward compatible change on a public interface, I'm hesitant to make this change in a bugfix release. I've been bitten by this too often.